requirements.txt

```diff
@@ -1,4 +1,4 @@
-django >= 1.7.0
+django >= 1.8.0
```

Comment:
djstopie is only used in yolacom and sitebuilderui so I think I'm going to update it to just support 1.11+.  Similar logic as with djyotest